applications/main/nfc/scenes/nfc_scene_delete.c

```diff
@@ -52,6 +52,7 @@ void nfc_scene_delete_on_enter(void* context) {
 
 bool nfc_scene_delete_on_event(void* context, SceneManagerEvent event) {
     Nfc* nfc = context;
+    furi_string_printf(nfc->dev->folder, "%s", NFC_APP_FOLDER);
```

Comment:
Looks like something that can be passed to allocator.

Reply:
That's fair, fixed now